### ISSUE TITLE
R_curl_version related error when installing curl package.

### ISSUE THREAD

**Issue by airbj31** [jeroenooms/curl#65]
```
version.c: In function ‘R_curl_version’:
version.c:15:43: error: ‘curl_version_info_data {aka const struct <anonymous>}’ has no member named ‘libssh_version’
   SET_VECTOR_ELT(list, 3, make_string(data->libssh_version));
                                           ^
version.c:4:24: note: in definition of macro ‘make_string’
 #define make_string(x) x ? Rf_mkString(x) : ScalarString(NA_STRING)
                        ^
version.c:15:43: error: ‘curl_version_info_data {aka const struct <anonymous>}’ has no member named ‘libssh_version’
   SET_VECTOR_ELT(list, 3, make_string(data->libssh_version));
                                           ^
version.c:4:40: note: in definition of macro ‘make_string’
 #define make_string(x) x ? Rf_mkString(x) : ScalarString(NA_STRING)
                                        ^
make: *** [version.o] Error 1
ERROR: compilation failed for package ‘curl’
* removing ‘/home/bjkim/local/R-latest/lib64/R/library/curl’
```
------------------------------------------------------------------------------------------

Hi, I got above error when installing curl package.
is there any way to solve this issue ?

I'm using centOS 5.9 / curl-devel-7.15.5-17.el5_9

thanks for your help.

**Comment by jeroenooms** [comment]
Wow that's really old. I would recommend you install a more recent version of libcurl.

**Comment by jeroenooms** [comment]
Can you try if the pervious version works:

```sh
wget https://cran.r-project.org/src/contrib/Archive/curl/curl_0.9.6.tar.gz
R CMD INSTALL curl_0.9.6.tar.gz
```

**Comment by airbj31** [comment]
thanks for giving me the information.

I solved the problem by installing libcurl-7.48 from source code :)

thank you !
